src/dotnet/APIView/APIViewWeb/HostedServices/ReviewBackgroundHostedService.cs

```diff
@@ -13,6 +13,7 @@ public class ReviewBackgroundHostedService : IHostedService, IDisposable
     {
         private bool _isDisabled = false;
         private ReviewManager _reviewManager;
+        private int _autoArchiveInactiveGracePeriod; // This is inactive duration in months
```

Comment:
Maybe `_autoArchiveInactiveGracePeriodMonths` since that's similar to the config key and then you don't need the comment?